.vscode/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+      {
+        "label": "compile",
+        "command": "npm",
+        "args": ["run", "compile"]
+      }
+    ]
+  }
```

Comment:
nit: Formatting in this file is incorrect.

Reply:
Fixed in d0b6d3b.